great-idea/index.html

```diff
@@ -1,23 +1,128 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <title>Great Idea!</title>
+    <title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link
+      href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
 
   <body>
+    <div class="container">
+      <nav class="nav">
+        <ul>
+          <li><a href="services.html">Services</a></li>
+          <li><a href="products.html">Product</a></li>
+          <li><a href="#">Vision</a></li>
+          <li><a href="#">Features</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a href="index.html">
+            <img
+              class="logo"
+              src="img/logo.png"
+              alt="Great Idea! Company logo."
+            /></a>
+          </li>
+        </ul>
+      </nav>
+    </div>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+    <div class="header">
+      <div class="header-text-container">
+        <h1 class="header-text">
+          Innovation <span class="new-line">On </span> Demand
+        </h1>
+        <button class="button">Get Started</button>
+      </div>
+
+      <img src="img/header-img.png" alt="Image of a code snippet." />
+    </div>
+
+    <div class="container">
+      <div class="section-2">
+        <h1 class="h1-text">Features</h1>
+        <p>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse
+          vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam
+          convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+      </div>
+
+      <div class="section-2">
+        <h1 class="h1-text">About</h1>
```

Comment:
I believe this is just because is the same code from the day before, but you should try to not use more the one `<h1>` per page. This will lead to better semantic HTML.